app/views/provider_management/external_users/confirmation.html.haml

```diff
@@ -0,0 +1,11 @@
+%h1.govuk-heading-xl
+  =t('.text')
+%h2.govuk-heading-m
+  =t('.disable_message')
+.govuk-button-group
+  = govuk_link_button_warning(t('.disable_user'), disable_provider_management_provider_external_user_path(@provider, @external_user))
+  = govuk_link_to(t('.cancel_button'), provider_management_provider_external_user_path(@provider, @external_user))
+
+
+
+
```

Comment:
- you may wish to wrap this content in a two thirds column
- break up code to make it easier to read
- Remove additional lines at end of file